src/lib/operators/join_hash.hpp

```diff
@@ -52,6 +52,19 @@ class JoinHash : public AbstractJoinOperator {
     size_t radix_bits{0};
     // Initially, the left input is the build side and the right side is the probe side.
     bool left_input_is_build_side{true};
+
+    // Due to the used Bloom filters, the number of actually joined tuples can significantly differ from the sizes of
+    // the input tables. To enable analyses of the Bloom filter efficiency, we store the number of values that were
+    // eventually materialized; i.e., "input_row_count - filtered_values_by_Bloom_filter".
+    size_t build_side_materialized_value_count{0};
```

Comment:
You said

> build_side_materialized_value_count does not clearly state what is means, I think.

Are you fine with it after all? If you are happy, I am happy. ;)

Reply:
Happy with it. I think the problem was that I misunderstood you and though you wanted to use the name for the size of the hash table.